promptsource/templates/generated_reviews_enth/templates.yaml

```
@@ -6,52 +6,62 @@ templates:
     jinja: Does "{{translation.en}}" seem like a positive review to you? ||| {{answer_choices[0
       if review_star<3 else 1]}}
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - AUC
+      - Accuracy
       original_task: true
-    name: sentiment_analysis_4
+    name: seem like a positive review
     reference: stsb_multi_mt_en
   95136948-3402-4bd4-8a69-1aa7b85461cc: !Template
     answer_choices: null
     id: 95136948-3402-4bd4-8a69-1aa7b85461cc
     jinja: 'Rate the positivity of this review: ({{"1"}} being the lowest and {{"5"}}
       the highest) "{{translation.en}}" ||| {{review_star}}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: true
+      metrics:
+      - AUC
+      - Accuracy
       original_task: true
-    name: sentiment_analysis_5
+    name: rate positive review
     reference: stsb_multi_mt
   ad12212f-a230-4750-a199-9791628856c4: !Template
     answer_choices: null
     id: ad12212f-a230-4750-a199-9791628856c4
     jinja: "How positive is the review \"{{translation.en}}\"? Give a score between\n\
       \      {{\"0\"}} and {{\"5\"}}. ||| {{review_star}}"
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: true
```

```suggestion
      choices_in_prompt: false
```
https://huggingface.slack.com/archives/C023Z96L7B4/p1635785857084900